src/back/shared/errors_catchit.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "errors.hpp"
+
+#include <userver/components/component_config.hpp>
+#include <userver/components/component_context.hpp>
+#include <userver/server/handlers/http_handler_json_base.hpp>
+#include <userver/utest/using_namespace_userver.hpp>
```

Comment:
Давай не инклудить то что не нужно, это может приводить к замедлению сборки.
```suggestion
#include <userver/formats/json/value.hpp>
#include <userver/server/http/http_request.hpp>
#include <userver/utest/using_namespace_userver.hpp>
```

Reply:
Готово (почистил везде в shared).